core/cont/inc/TList.h

```diff
@@ -85,10 +85,15 @@ friend  class TListIter;
    void              AddLast(TObject *obj) override;
    virtual void      AddLast(TObject *obj, Option_t *opt);
    void              AddAt(TObject *obj, Int_t idx) override;
+   void              AddAt(TObject *obj, Int_t idx, Option_t *option);
```

Comment:
A challenge is that those routines are non-virtual while their counterpart without option are virtual.  This virtuality is used by `THashList` and possibly (maybe) by other user classes.  As-is, if one call this signature on a `THashList`, the state of the container will be invalid/incorrect.